src/oaklib/implementations/simpleobo/simple_obo_implementation.py

```diff
@@ -861,7 +861,7 @@ def _tv_dict(stanza: Stanza) -> Dict[str, List[str]]:
                         id=_id(), about_node=t1id, new_value=td2, old_value=td1
                     )
                 elif vals1:
-                    yield kgcl.RemoveTextDefinition(id=_id(), about_node=t1id)
+                    yield kgcl.RemoveTextDefinition(id=_id(), about_node=t1id, old_value=vals1.pop())
```

Comment:
don't mutate, lookup instead! otherwise all good, thx!

Reply:
fixed. It's already there in `td1`